qtdialogs/qtdefines.py

```diff
@@ -621,40 +921,43 @@ def createBitmap(imgMtrx2D, writeToFile=-1, returnBinary=True):
       except:
          return False
 
-
-
 def selectFileForQLineEdit(parent, qObj, title="Select File", existing=False, \
```

Comment:
this is a perfect candidate to fix for formatting, the arg alignment is vomit inducing and the trailing `\` is superfluous in py3